screens/HomeScreen.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { FC } from 'react';
+import { StyleSheet, Text, TouchableOpacity, View, ScrollView, Image, Button } from 'react-native';
+import { useSelector } from 'react-redux';
+import colors from '../assets/colors/colors';
+import Coin from '../components/Coin';
+import { Crypto } from '../interfaces/types';
+import { RootState } from '../store';
+
+const HomeScreen: FC = ({ navigation }: any) => {
+
+  const list = useSelector((state: RootState) => state.cripto.cryptoList)
+
+  return (
+    <ScrollView style={styles.homeContainer}>
+
+      {/* Header */}
+
+      <View style={styles.headerContainer}>
+        <Text style={styles.headerTitle}>CryptoTracker Pro</Text>
+        <Image source={require('../assets/images/girl.png')} />
+      </View>
+
+      {/* Crypto Coins */}
+
+      <View>
+        {list?.map((item: Crypto) => (
```

Comment:
[CRITICAL] Never use .map() to render a list with unknown length; this can have CRITICAL performance issues on any scalable app. Also, this view is not scrollable. Please switch to using a FlatList.